### PR TITLE
add a try-catch to prevent admin from going down with graph

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner @leogrim not sure what you guys think of this, but it's something that @gratimax and Tamila have asked for since graph has been so flaky. this way we report and just return an empty list.

### CODE REVIEW

**Comment by andrewconner** [comment]
Down with me, but @leogrim may know better

**Comment by maxov** [comment]
This is great :+1: I also proposed returning an `Either[RecoError, List[Reco]]` for some recommended type `Reco` (like `User` or `Organization`) and a series of types representing errors `RecoError`. This way we could see in the admin that the graph query failed with some sort of error instead of a plain list. An empty list might imply that the graph simply couldn't find and recommendations, while an error message visibly says that there was something wrong with fetching the data.

**Comment by camhashemi** [comment]
ah, @gratimax. that's better.

**Comment by leogrim** [comment]
I'm down, just like the Graph.

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala [hunk 1]
```diff
@@ -254,7 +255,11 @@ class AdminUserController @Inject() (
     val userId = user.id.get
     val abookInfoF = abookClient.getABookInfos(userId)
     val econtactCountF = abookClient.getEContactCount(userId)
-    val fOrgRecos = abookClient.getOrganizationRecommendationsForUser(user.id.get, offset = 0, limit = 5)
+    val fOrgRecos = try {
+      abookClient.getOrganizationRecommendationsForUser(user.id.get, offset = 0, limit = 5)
```
Comment (maxov) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala:259]: I just realized, there's a method `fallbackTo` on Future that accomplishes this

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala:259]: We could use that, but we still want to call `airbrake.notify(ex)`.

**Comment by camhashemi** [comment]
@gratimax I'm going to merge this to get it out there, and we can add error messages later.

**Pull request merged**